runtime/src/launch/aprun/aprun-utils.c

```diff
@@ -211,10 +230,11 @@ static char _Nbuf[16];
 static char _jbuf[16];
 char** chpl_create_aprun_cmd(int argc, char* argv[],
```

Comment:
Not a comment on this PR but just in general: how I wish we hadn't had so much pushback against making incremental improvements to the launchers while maintaining them.  Every time I gaze upon this function I'm tempted to weep.